src/App.js

```diff
@@ -1,17 +1,40 @@
 import "./App.scss";
-import Header from "./components/Header/Header";
+import Search from "./components/Search/Search";
 import Footer from "./components/Footer/Footer";
+import CurrentWeather from "./components/CurrentWeather/CurrentWeather";
+import FutureWeather from "./components/FutureWeather/FutureWeather";
+// import WeatherInfo from "./weatherInfo.json"
+
+import { useState } from "react";
 
 //configs
 const siteTitle = process.env.REACT_APP_SITE_TITLE ?? "CYF Weather";
 
 function App() {
+
+   // A state variable to hold weather Array
+    const [weatherInfo, setWeatherInfo] = useState([])  //Initialized as an Empty Array
+console.log('This is Weather Info: ', weatherInfo);
+  if(weatherInfo.length > 0){
+    let infWeather =  weatherInfo[0].weather[0].id 
+ }
+//  <sup>o</sup>
   return (
     <div className="App">
-      <Header title={siteTitle} />
+       <Search title={siteTitle} setWeatherInfo={setWeatherInfo} />
+       <div className="weather-container "
+      //  {
+      //   true ? style={{backgroundColor: "This"}}: 
+      //  }
+       >
+        <CurrentWeather weatherInfo={weatherInfo} />
+        <FutureWeather weatherInfo={weatherInfo} />
```

Comment:
Do you need to pass ALL the data to both these components? It's always best to pass down just what a component needs. 
In this case: `CurrentWeather` only needs one item from the array of weather in the API response. And `FutureWeather` needs 7 items (according to the design which is our source of truth here), so you need to make a small array of 7 items here, and pass down that small array to `FutureWeather` instead of the whole thing